docker/thehive/thehive/entrypoint.py

```diff
@@ -36,6 +40,29 @@
 THEHIVE_ORGADMIN_PWD = os.environ["DEFAULT_ORGADMIN_PWD"]
 
 
+def is_initialized() -> bool:
+    if not path.isfile(CHECKPOINT_LOCATION):
+        logging.info("TheHive and Cortex not initialized yet")
+        return False
+    with open(CHECKPOINT_LOCATION, "r") as checkpoint_file:
+        status = checkpoint_file.read()
+        if status != "COMPLETED":
+            raise Exception(INCONSITENT_STATE_ERR)
```

Comment:
we also make the entrypoint script easier to debug by explicitly failing if the init is only partially completed